addons/web/static/src/views/calendar/calendar_renderer.js

```diff
@@ -24,36 +24,58 @@ export class CalendarRenderer extends Component {
         onSquareSelection: Function,
         cleanSquareSelection: Function,
     };
+    setup() {
+        this.LONG_TOUCH_THRESHOLD = 400;
```

Comment:
Could we harmonize this constant with the `longPressDelay` props ? I feel like both are the same concept but expressed differently.
Maybe I missed something... 🙈